mayastor/src/bin/mayastor.rs

```diff
@@ -52,6 +60,8 @@ fn start_tokio_runtime(args: &MayastorCliArgs) {
     });
 }
 
+const VERSION_STR: &str = "0.9.0 C2.RC2";
```

Comment:
This will be useful :+1: 
Hmm we could actually read the cargo version you just updated on the Cargo.toml from nix and use it though that wouldn't be very unique. Maybe the jenkins pipeline could help.
Btw, @GlennBullingham do we have an "official" way of generating release images from Jenkins?

Reply:
Yeah, we should do this automatically within nix-build. But, I would need some time to actually look into that. Likely we want to create a file in `$out` to write down the commit and version automatically.